pkgs/applications/networking/remote/citrix-workspace/generic.nix

```diff
@@ -209,6 +209,9 @@ stdenv.mkDerivation rec {
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
     platforms = platforms.linux;
     maintainers = with maintainers; [ pmenke michaeladler ];
+    knownVulnerabilities = lib.optionals (lib.versionOlder version "23.02") [
+      "CVE-2023-24486"
```

Comment:
Would be best to additionally give the user running into this some reading material to judge the situation themselves e.g., the advisory url.

Reply:
Sure, I've added it